packages/orchestration/README.md

```diff
@@ -17,16 +17,145 @@ $ npm install @sap-ai-sdk/orchestration
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## Orchestration Client
 
-<!-- Details to be added -->
+The orchestration client provides essential features like templating and content filtering, which are often required in business AI scenarios:
 
-## Support, Feedback, Contribution
+- **Templating** allows composing prompts with placeholders that can be filled during a chat completion request.
+- **Content filtering** lets you restrict the type of content sent to or received from a generative AI model.
+
+To interact with generative AI models compatible with the orchestration service, use the orchestration client.
+More details about orchestration are available [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
+
+To utilize orchestration capabilities, you need to create a deployment.
+Once the deployment is complete, you'll have a `deploymentUrl` to access the orchestration service.
+For details, see [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-orchestration).
+
+### Prerequisites
+
+Ensure the following are in place:
+
+- At least one orchestration-compatible deployment for a generative AI model is running.
+  - You can use the [`DeploymentApi`](../ai-api/README.md#deploymentapi) from `@sap-ai-sdk/ai-api` to deploy a model to the SAP generative AI hub. For more information, see [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core).
+- A deployment for orchestration is created. For details, see [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-orchestration).
+- The `@sap-ai-sdk/orchestration` package is installed in your project.
+
+### Orchestration Client
+
+Use the `OrchestrationClient` to configure different modules like templating and content filtering along with sending chat completion requests to an orchestration compatible generative AI model.
+
+You can pass configuration information like for example the LLM to be used for inference and template, as a parameter to the client.
```

Comment:
[pp] I don't find this sentence very helpful. Maybe we could even just remove it?
```suggestion
```